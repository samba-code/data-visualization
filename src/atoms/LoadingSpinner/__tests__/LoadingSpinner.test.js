import React from "react";
import { render } from "enzyme";

import LoadingSpinner from "../LoadingSpinner";

describe("LoadingSpinner", () => {
  it("should match snapshot", () => {
    const wrapper = render(<LoadingSpinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
