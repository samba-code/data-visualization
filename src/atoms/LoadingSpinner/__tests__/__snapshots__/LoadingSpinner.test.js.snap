// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner should match snapshot 1`] = `
<div
  class="LoadingSpinner__StyledLoading-sc-17pj9ki-0 jNAaLu"
>
  <span
    class="css-0"
  >
    <span
      class="css-129peub"
    />
    <span
      class="css-prnvxp"
    />
    <span
      class="css-8el4s5"
    />
    <span
      class="css-1hf2v34"
    />
    <span
      class="css-tx0a7l"
    />
  </span>
  <p
    class="LoadingSpinner__LoadingText-sc-17pj9ki-1 iQTncG"
  >
    Loading data
  </p>
</div>
`;
